robottelo/vm.py

```diff
@@ -162,10 +161,9 @@ def create(self):
                 u'Failed to run snap-guest: {0}'.format(result.stderr))
 
         # Give some time to machine boot
-        time.sleep(60)
-
         result = ssh.command(
-            u'ping -c 1 {0}.local'.format(self._target_image),
+            u'for i in {{1..60}}; do ping -c1 {0}.local && break; sleep 1;'
+            u' done; echo $?'.format(self._target_image),
```

Comment:
this will move the timeout logic onto  the hypervisor itself, so it keeps pinging the host for availability __until ready__ or timeout passes. This speeds up things a bit since we don't wait a constant time (full minute) everytime. After testing, the host typically boots up whithin ~30seconds.